08-Secure by default/Secure Templates/ARM/Synapse.json

```diff
@@ -0,0 +1,157 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "synapseWorkspaceName":{
+            "type": "string",
+            "metadata": {
+                "description": "Synapse workspace name."
+            }
+        },
+        "dataLakeStorageAccountName":{
+            "type": "string",
+            "metadata":{
+                "description": "Url of the data lake storage account which we need to associate with synapse workspace."
+            }
+        },
+        "dataLakeStorageFileSystem":{
+            "type": "string",
+            "metadata":{
+                "description": "File system name to associate with data lake storage and synapse workspace."
+            }
+        },
+        "sqlAdminUser":{
+            "type": "string",
+            "metadata":{
+                "description": "Sql admin user."
+            }
+        },
+        "sqlAdminUserPassword":{
+            "type": "string",
+            "metadata":{
+                "description": "Password for sql admin user."
+            }
+        },
+        "storageAccountId":{
+            "type": "string",
+            "metadata":{
+                "description": "Storage account id to which the diagnostic logs and metrices will be sent."
+            }
+        }
+    },
+    "functions": [],
+    "variables": {},
+    "resources": [
+        {
+            "type": "Microsoft.Synapse/workspaces",
+            "apiVersion": "2019-06-01-preview",
```

Comment:
we don't have stable version to check the same?

Reply:
Changed to 2021-06-01